optimum/intel/openvino/configuration.py

```diff
@@ -198,6 +214,9 @@ def __init__(
         self.all_layers = all_layers
         self.sensitivity_metric = sensitivity_metric
         self.quant_method = quant_method
+        self.scale_estimation = scale_estimation
+        if awq:
+            self.quant_method = QuantizationMethod.AWQ
```

Comment:
I'd prefer to only have one path to enable awq (currently we have both `quant_method` and `awq` arguments), what do you think about something like `OVWeightQuantizationConfig(quant_method="awq")` ?

Reply:
@echarlaix Ah, I thought you meant it's fine keeping both. Ok, removed `awq` argument.

Reply:
thanks a lot @nikita-savelyevv